src/main/io/cms_blackbox.c

```diff
@@ -0,0 +1,77 @@
+//
+// CMS things for blackbox and flashfs.
+// Should be part of blackbox.c (or new blackbox/blackbox_cms.c) and io/flashfs.c
+//
+#include <stdbool.h>
```

Comment:
I believe these are resurrection of files moved to cms directory; I was working with CMS and SmartAudio simultaneously.